### PR TITLE
Development instances

### LINKED ISSUE #130: Explore options for dev instances
**Is your feature request related to a problem? Please describe.**
Currently we have a series of steps to hook a dev instance of `thunderbird` into the `birdhouse-deploy` project. We want to simplify that process.

**Describe the solution you'd like**
We may be able to just lightly modify the basic `docker-compose.yml` and use built-in commands.

**Describe alternatives you've considered**
Alternatively we could set up a "dev" `pavics-component.

### PR DESCRIPTION
**Pull request by nikola-rados**
Some of the docker components have been adjusted to allow for `docker-compose` to be used on `dev03`. There is an instance running, run `make test-notebooks-dev` to test.

Resolves #130.

### CODE REVIEW

**Review by cairosanders — APPROVED** [review]
:+1:

File: docs/source/notebooks/wps_generate_climos_demo.ipynb [hunk 1]
```diff
@@ -16,7 +16,7 @@
   },
```
Comment (cairosanders) [inline docs/source/notebooks/wps_generate_climos_demo.ipynb:16]: <p>Oh actually, you probably need to change this back</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pacificclimate/thunderbird/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (nikola-rados) [reply docs/source/notebooks/wps_generate_climos_demo.ipynb:16]: <p>Good catch! </p>

**Review by sum1lim — APPROVED** [review]
LGTM!

**Pull request merged**